src/Plugins/SimpleCommands/CommandsViewModel.cs

```diff
@@ -17,6 +17,15 @@ public CommandsViewModel(ICommandsSet commandsSet)
             Icon = new PackIconFontAwesome { Kind = PackIconFontAwesomeKind.ExclamationSolid };
             Label = "Commands";
             ToolTip = "Custom Commands";
+            IsOption = false;
+            Content = new CommandsView { DataContext = this };
```

Comment:
Even if we DI this, we have to avoid a circular dependency, since this does need the data context set. We'll need to get `Content` off of the `IMenuItemViewModel` to get rid of this circle.

Reply:
The issue I had with DI on view models was switching to a situation where no ViewModel held a direct reference to a form or user control. In essence, it shouldn't know that WPF is the UI tech.